resources/operator.yaml

```diff
@@ -20,14 +20,16 @@ spec:
           - /usr/local/bin/ao-logs
           - /tmp/ansible-operator/runner
           - stdout
-          image: quay.io/benchmark-operator/benchmark-operator:master
+          #image: quay.io/benchmark-operator/benchmark-operator:master
+          image: quay.io/mkarg/benchmark-operator:testing
```

Comment:
Can we flip this back to the "normal" benchmark-operator account? Same with the below

Reply:
Done